semanticcpg/src/main/scala/io/shiftleft/semanticcpg/language/types/expressions/Call.scala

```diff
@@ -12,8 +12,8 @@ import io.shiftleft.semanticcpg.language.types.structure.{Member, Method, Method
 /**
   A call site
   */
-class Call(val wrapped: NodeSteps[nodes.Call]) extends EvalTypeAccessors[nodes.Call] {
-  override val raw: GremlinScala[nodes.Call] = wrapped.raw
+class Call(val wrapped: NodeSteps[nodes.Call]) extends AnyVal {
+  def raw: GremlinScala[nodes.Call] = wrapped.raw
```

Comment:
Make this private. Otherwise we only have a `raw` step for some node types depending on the fact whether we use such a short hand or not.
I have seen this `raw` method not being private in other places too. Please correct those also.

Reply:
agreed, done in 05a8965f1af48eadf4b9b5264c8d842d66c2157f